packages/nextcloud/test/api/webdav/webdav_test.dart

```diff
@@ -23,6 +26,35 @@ class _FileMock extends Mock implements File {}
 
 class _FileStatMock extends Mock implements FileStat {}
 
+List<String> _getMultistatusPropNames(XmlElement root) {
+  return root.firstElementChild!.childElements
+      .singleWhere(
+        (node) =>
+            node.name.local == 'propstat' &&
+            node.childElements.singleWhere((node) => node.name.local == 'status').innerText.contains('200 OK'),
+      )
+      .childElements
+      .singleWhere((node) => node.name.local == 'prop')
+      .childElements
+      .map((el) => '{${el.name.namespaceUri}}${el.name.local}')
+      .toList()
+      .sorted();
+}
+
+Future<void> _expectAllPropsAreParsed(NextcloudTester tester, WebDavMultistatus result, PathUri uri) async {
+  final parsedProps = _getMultistatusPropNames(result.toXmlElement(namespaces: namespaces));
+  expect(parsedProps, isNotEmpty);
+
+  final streamedResponse = await tester.client.webdav.httpClient.send(tester.client.webdav.propfind_Request(uri));
```

Comment:
Do these calls need to be in the fixtures?
I'd rather have it not included.

Reply:
No, we can exclude them

Reply:
Then I'd love to have it like that.